front/src/views/Home.vue

```diff
@@ -1,5 +1,6 @@
 <template>
   <div class="home">
+    {{user.uid}}
```

Comment:
Do we keep this ? Or in a near futur we display user name with a awesome welcome message ?